src/getCoinCombination.js

```diff
@@ -5,7 +5,12 @@
  *
  * @returns {number[]}
  */
+
 function getCoinCombination(cents) {
+  if (cents < 0) {
+    throw new Error('Not valid argument');
```

Comment:
The error message 'Not valid argument' could be more descriptive. Consider specifying that the argument must be a non-negative number.